modules/dynamic-provisioning-azure-disk-definition.adoc

```diff
@@ -15,29 +15,22 @@ metadata:
   name: slow
 provisioner: kubernetes.io/azure-disk
 parameters:
-  storageAccount: azure_storage_account_name  <1>
-  storageaccounttype: Standard_LRS  <2>
-  kind: Dedicated  <3>
+  storageAccount: azure_storage_account_name <1>
+  storageaccounttype: Standard_LRS <2>
+  kind: Managed <3>
 ----
-<1> Azure storage account name. This must reside in the same resource
-group as the cluster. If a storage account is specified, the `location`
-is ignored. If a storage account is not specified, a new storage
-account gets created in the same resource group as the cluster. If you
-are specifying a `storageAccount`, the value for `kind` must be `Dedicated`.
-<2> Azure storage account SKU tier. Default is empty. Note that Premium
-VMs can attach both `Standard_LRS` and `Premium_LRS` disks, Standard VMs
-can only attach `Standard_LRS` disks, Managed VMs can only attach
-managed disks, and unmanaged VMs can only attach unmanaged disks.
-<3> Possible values are `Shared` (default), `Dedicated`, and `Managed`.
+<1> Azure storage account name. This must reside in the same resource group as the cluster. If a storage account is specified, the `location` is ignored. If a storage account is not specified, a new storage account gets created in the same resource group as the cluster. If you are specifying a `storageAccount`, the value for `kind` must be `Dedicated`.
```

Comment:
With the new storageclass, seems this is unnecessary.

Reply:
Removed